spec/TotalCalculatorSpec.js

```diff
@@ -0,0 +1,27 @@
+describe('TotalCalculator', function() {
+  var totalCalculator;
+
+  describe('calculates discounts', function() {
+    it('5% off orders over £50', function() {
+      totalCalculator = new TotalCalculator();
+      totalCalculator.subTotal = 50.01;
+      totalCalculator.orderedItems = [];
+      totalCalculator.detailsList = hipsterCoffeeDetails;
+      expect(totalCalculator.calculateTotalBill()).toEqual(47.51);
```

Comment:
You've encapsulated the tax calculation into this object, so you probably shouldn't have to test it directly in the object that uses it